smarts/core/sensors.py

```diff
@@ -17,7 +17,9 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
+from enum import Enum
 import logging
+from smarts.core.traffic_history import TrafficHistory
 import time
 from collections import deque, namedtuple
```

Comment:
Ordering `import` and `from` statements alphabetically improves code presentation.